package.json

```diff
@@ -96,6 +96,7 @@
     "codecov": "^3.7.2",
     "cucumber": "^6.0.5",
     "eslint": "^7.6.0",
+    "js-yaml": "^3.14.0",
```

Comment:
Did you remove the former package `jsonToYaml` ?

Reply:
Nop, `jsonToYaml` is only for JSON to YAML conversion but I need YAML to JSON conversion in unit tests to performs a few checks (this is why I add it in the `devDependencies` only)